PendingReleaseNotes

```diff
@@ -3,8 +3,8 @@
 * New bluestore_rocksdb_options_annex config parameter. Complements
   bluestore_rocksdb_options and allows setting rocksdb options without repeating
   the existing defaults.
-* The cephfs addes two new CDentry tags, 'I' --> 'i' and 'L' --> 'l', and
-  on-RADOS metadata is no longer backwards compatible after upgraded to Pacific
+* The cephfs adds two new CDentry tags, 'I' --> 'i' and 'L' --> 'l', and
+  on-RADOS metadata is no longer backwards-compatible after upgrade to Pacific
   or a later release.
```

Comment:
```suggestion
* The MDS in Pacific makes backwards-incompatible changes to the on-RADOS metadata structures which prevent a downgrade to older releases (Octopus and older).
```

Reply:
I've replaced this with Patrick's suggestion.